css/style.css

```diff
@@ -0,0 +1,25 @@
+form {
+  background: rgba(27, 27, 50, 0.8);
+  border-radius: 0.25rem;
+  padding: 2.5rem
+;
+}
+
+body {
+  color: white !important;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background: #00008b;
+  background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ), url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible using the Template provided. I strongly recommend do changes on the labels box properties(margin, padding, etc) using their id or adding a class and then style them so the labels get closer to their respective input to avoid misunderstandings

![image](https://user-images.githubusercontent.com/82236725/150269657-db5dedd4-aa01-48e7-8392-bebb2310267d.png)